core/src/test/scala/unit/kafka/admin/TopicCommandTest.scala

```diff
@@ -188,4 +189,103 @@ class TopicCommandTest extends ZooKeeperTestHarness with Logging with RackAwareT
     }
     checkReplicaDistribution(assignment, rackInfo, rackInfo.size, alteredNumPartitions, replicationFactor)
   }
+
+  def testDescribePartitions() {
```

Comment:
Thanks for the tests! They are obviously sensitive to any change in formatting.

Is the intention to guarantee that the formatting will never change?